packages/fxa-settings/README.md

```diff
@@ -9,6 +9,75 @@ This documentation is up to date as of 2020-08-03.
 - `yarn test` to run unit tests
 - `yarn storybook` to open Storybook
 
+### GQL and REST API Calls
+
+FxA Settings communicates primarily with the FxA GraphQL API through use of [Apollo Client](https://www.apollographql.com/docs/react/) to indirectly interact with the `fxa-auth-server`. Requests that simply retrieve data are called queries and all other interaction requests are called mutations. See the [GQL documentation](https://graphql.org/learn/) to learn more. See (the documentation for `fxa-graphql-api`)[https://github.com/mozilla/fxa/tree/main/packages/fxa-graphql-api)] to connect to the playground, a place to view the API docs and schema, and to write and test queries and mutations before using them in a component.
+
+While most API calls can be performed with GQL, there are a few calls that must directly communicate with the auth server (for now) for security reasons. To make REST API calls to the auth server, use the `useAuth` and `useAwait` hooks.
```

Comment:
TIL `useAuth`! I am missing all these fun new things going by :)

Reply:
It hasn't been used yet, but Danny made a (now closed) "change password sketch" PR where he's got an example use!